src/Directory.Build.props

```diff
@@ -5,7 +5,11 @@
     <Authors>OpenSearch Project and contributors</Authors>
     <Copyright>OpenSearch</Copyright>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <RepositoryUrl>https://github.com/opensearch-project/opensearch-net-abstractions</RepositoryUrl>
+    <RepositoryUrl>https://github.com/Bit-Quill/opensearch-net-abstractions-1</RepositoryUrl>
+    <!-- Use the value below when merging to upstream.
```

Comment:
You've obviously tested this. Do we put in the upstream version for fear of missing this change later?

Reply:
@guiangumpac  brought up the same point -- I changed it to point to opensearch-project.